nixos/modules/services/misc/overseerr.nix

```diff
@@ -0,0 +1,64 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.overseerr;
+in
+{
+  meta.maintainers = [ maintainers.caarlos0 ];
+
+  options.services.overseerr = {
+    enable = mkEnableOption (mdDoc ''
+      Request management and media discovery tool for the Plex ecosystem
+    '');
+
+    openFirewall = mkOption {
```

Comment:
why not use `mkEnableOption`?

Reply:
I saw that some other services have this option too... I can change it though, either way is fine to me.

I think the diff is that if you only use it locally, there's no need to open the port